paasta_tools/kubernetes_tools.py

```diff
@@ -1516,8 +1516,63 @@ def get_pod_volumes(
         if crypto_volume:
             pod_volumes.append(crypto_volume)
 
+        database_credentials_secrets_volumes = (
+            self.get_database_credentials_secrets_volumes()
+        )
+        if database_credentials_secrets_volumes:
+            # note an array is returned so we extend vs append
+            pod_volumes.extend(database_credentials_secrets_volumes)
+
         return pod_volumes
 
+    def get_database_credentials(self) -> List[str]:
+        database_credentials = self.config_dict.get("database_credentials", {})
+        return database_credentials
+
+    def get_database_credentials_secret_name(self, dstore, credential) -> str:
```

Comment:
I'm open to it! though we've unfortunately hardcoded it in multiple places, I'll have to refer to flopez on updating this

eg vault paths, yelp_conn code, terraform stuff?

Reply:
i mentioned this in slack, but just scribing here as well: i was referring more to the variable name - but it might be more descriptive to also have the actual paths be something more verbose :)

Reply:
++ we changed everything to datastore where applicable. makes more sense!